php/php.editor/src/org/netbeans/modules/php/editor/csl/SemanticAnalysis.java

```diff
@@ -819,6 +820,12 @@ public void visit(ConstantDeclaration node) {
                     if (!isPrivate || parentNode instanceof TraitDeclaration) {
                         addColoringForNode(identifier, coloring);
                     } else {
+                        // NOTE: private constants, methods, and fields may be used in traits
+                        // currently, we don't check traits (if there is no performance problem, should check them if possible)
+                        // an enum is handled as a "final" class in PHP
+                        // so, virtually, "protected" is "private"
+                        // however, as wrote above, it may be used in traits
```

Comment:
```suggestion
                        // however, as written above, it may be used in traits
```

Reply:
@junichi11 Please, let me know if you plan to fix this typo. I can merge it as it is now, of course ☺️

Reply:
Will fix it :) Thanks!

Reply:
OK, so I wait for your commit and CI. Thanks!